src/qiskit_iqm/iqm_job.py

```diff
@@ -62,7 +63,7 @@ def _format_measurement_results(
         measurement_results: CircuitMeasurementResults,
         shape: tuple[int, int]
     ) -> list[str]:
-        formatted_results = {}
+        formatted_results:dict[int, np.ndarray] = {}
```

Comment:
Is this a linter suggestion to glue symbols around the colon?

Reply:
No, but it doesn't care one way or another. But I've updated it anyway, let's be consistent.